src/models/category.js

```
@@ -0,0 +1,34 @@
+const { Schema, model } = require('mongoose');
```

ESLint

[error] Extra semicolon.

(semi)